corehq/apps/export/models/new.py

```diff
@@ -792,6 +800,76 @@ def _new_from_schema(cls, schema):
         )
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = SMS_EXPORT
```

Comment:
not sure if this is correct here. `identifier` is used to specify the `xmlns` or `case_type` in an export. i think it may be Ok to have this be `None` if we don't need to further drilldown in the export